### ISSUE TITLE
Track version number using Docker's ARG instruction for Dockerfiles

### ISSUE THREAD

**Issue by kurtwheeler** [AlexsLemonade/refinebio#530]
### Context

After running the Zebrafish Crunch @cgreene requested we enable sentry's releases feature like so:

```
RAVEN_CONFIG = {
    'dsn': 'REDACTED',
    # If you are using git, you can also automatically configure the
    # release based on the git info.
    'release': raven.fetch_git_sha(os.path.abspath(os.pardir)),
}
```

### Problem or idea

The problem is that we don't run our project from a git repo. We copy the relevant code into a Docker image, then run that. The git hash is not available from within the docker container.

We can put the git hash into the docker container using the [ARG instruction](https://docs.docker.com/engine/reference/builder/#arg). However if we're going to do that, we may as well just use the same instruction to manage our version number.

This would solve a couple other issues I've been concerned about for a bit:
* We have `_version.py` files in our project we used at one point, I'm not even sure if they're used anymore but because they're in the code they haven't been getting updated. Keeping the version actually in the code means that in order to do a release, we need to update the code first.
* We specify docker images in our Nomad Job Specifications. However we are always using the `latest` tag. This means that if a new image is pushed, the latest tag will probably update and we could end up running new code without running a deployment. Specifying exactly the tag we want to run will mean that unless we repush to that tag, the image won't change under us. We have code in our deploy process that prevents us from overwriting existing docker images for this reason.

### Solution or next step

In our code that updates the docker images, we should pass the version as an ARG to `docker build` and use that for the sentry release, our system version number, and then when we format the Nomad Job Spec templates, specify that version as the tag for the docker image.

### New Issue Checklist

- [x] The title is short and descriptive.
- [x] You have explained the context that led you to write this issue.
- [x] You have reported a problem or idea.
- [x] You have proposed a solution or next step.

**Comment by jaclyn-taroni** [comment]
Closed by #606 @kurtwheeler ?

**Comment by kurtwheeler** [comment]
Yes!
